apps/dashboard/src/routes/dashboard/+page.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+	import { redirect } from '@sveltejs/kit';
+	import { modalStore } from '@skeletonlabs/skeleton';
+	import type { ModalSettings } from '@skeletonlabs/skeleton';
+	import CreateURLPrefixModal from './CreateURLPrefixModal.svelte';
+	import { goto } from '$app/navigation';
+
+	const urlCreationModal: ModalSettings = {
+		type: 'component',
+		component: { ref: CreateURLPrefixModal },
+		response: async (url: string) => {
+			await fetch('/dashboard/', {
+				method: 'POST',
+				body: JSON.stringify({ url: url }),
+				headers: { 'content-type': 'application/json' }
+			});
+
+			goto('/dashboard/link');
```

Comment:
Once the user has created their prefix URL, we want to redirect them to the page where they can create suffix links.

1. What data do people typically pass back to notify that the request was successful?
2. Any thoughts on route structure, should it be /dashboard/{prefix_url}?

Reply:
1. A JSON object, {success:true} is fine, then we can add more fields if needed.
2. Any API calls I suggest we put on /api/ as the base path, then use REST best practices.

For page routing, I was thinking the dashboard is the homepage (/), which would display your most recent links (all for now maybe), and would allow you to create more links.

Reply:
Move post call into this file